src/Micrometa/Domain/Item/PropertyList.php

```diff
@@ -122,6 +122,7 @@ public function next(): void
      *
      * @return mixed IRI key
      */
+    #[\ReturnTypeWillChange]
```

Comment:
This should raise the minimum php version to 8

Reply:
> As a remedy, this RFC proposes to add a ReturnTypeWillChange attribute which could be used to suppress the related E_DEPRECATED notices. Thanks to the backward compatible syntax of attributes, this can be done in code which is compatible with PHP 7 and below.

https://wiki.php.net/rfc/internal_method_return_types

Reply:
Oh thanks I didn't realize they were backwards compatible. Awesome 👍